types/api/v1/image_types.go

```diff
@@ -37,6 +37,7 @@ type ImageSpec struct {
 	ID            string      `json:"id,omitempty"`
 	Layers        []Layer     `json:"layers,omitempty"`
 	SealerVersion string      `json:"sealer_version,omitempty"`
+	ClusterImage  string      `json:"cluster_image,omitempty"`
```

Comment:
Please add more annotation for this added field. Spec struct is very the foundation part of sealer, we should make it explicit as possible. Add the motivation description between code line.

Back to theme, you wish to add the cluster runtime type, such as k0s, k3s, and k8s. Then I think adding a field of `ClusterRuntime` from https://github.com/sealerio/sealer/issues/1549#issuecomment-1173291475 is better.